src/panels/lovelace/entity-rows/hui-toggle-entity-row.ts

```diff
@@ -54,7 +55,9 @@ class HuiToggleEntityRow extends LitElement implements LovelaceRow {
 
     return html`
       <hui-generic-entity-row .hass=${this.hass} .config=${this._config}>
-        ${stateObj.state === "on" || stateObj.state === "off"
+        ${stateObj.state === "on" ||
+        stateObj.state === "off" ||
+        stateObj.state === UNAVAILABLE
```

Comment:
Not sure if this is needed, can it be another state that we would want to show as text?

Reply:
`UNKNOWN`?